src/algebraic_geometry/prime_spectrum.lean

```diff
@@ -261,6 +261,10 @@ lemma zero_locus_Union {ι : Sort*} (s : ι → set R) :
   zero_locus (⋃ i, s i) = (⋂ i, zero_locus (s i)) :=
 (gc_set R).l_supr
 
+lemma zero_locus_bUnion (s : set (set R)) :
+  zero_locus (⋃ s' ∈ s, s' : set R) = ⋂ s' ∈ s, zero_locus s' :=
+by rw [set.bUnion_eq_Union, zero_locus_Union, set.bInter_eq_Inter]
```

Comment:
The proof I was looking for is:
```suggestion
def zero_locus_bUnion (s : set (set R)) :
  zero_locus (⋃ s' ∈ s, s' : set R) = ⋂ s' ∈ s, zero_locus s' :=
by { have := (gc_set R).l_Sup, rw Sup_eq_supr at this, exact this }
```
but unless you can come up with a way to golf that, it's not better.